### PR TITLE
Improvement on JS compat: class inheritance documentation

### PR DESCRIPTION
**Pull request by vrescobar**
Improve class inheritance documentation regarding inheritance JS objects and extending them in fable in a type safe manner

fixes #2289 (at the fable compiler issue tracker)

### CODE REVIEW

**Comment by vrescobar** [comment]
Related Issue on a different repo:

https://github.com/fable-compiler/Fable/issues/2289

Please close the issue when this PR is merged.

File: docsrc/communicate/js-from-fable.md [hunk 1]
```diff
@@ -61,6 +61,28 @@ If the value is globally accessible in JS, you can use the `Global` attribute wi
  let [<Global>] console: JS.Console = jsNative
 ```
 
+
+#### OOP Class definition and inheritance
+
+Assuming we need to import an ES6 class or even a typescript class from which we only know the types, it is advised to attacj it to a class with an abstract member defining the types and a default implementation which points to the jsNative reference:
```
Comment (MangelMaxime) [inline docsrc/communicate/js-from-fable.md:67]: There is a typoe at: `attacj `

I think adding jsNative  betwing quotes can help better understand that's it's code like that `jsNative`

File: docsrc/communicate/js-from-fable.md [hunk 2]
```diff
@@ -156,6 +178,7 @@ Canvas.drawSmiley "smiley"
 
 The result would be the same, but the philosophy is slightly different. That's basically up to you to make a choice 😉
 
+
```
Comment (MangelMaxime) [inline docsrc/communicate/js-from-fable.md:181]: Can you please delete this empty line

**Comment by vrescobar** [comment]
Thanks for the review Corrected with typo, suggestion, and newline. Feel free to amend the commit 👍

**Comment by alfonsogarciacaro** [comment]
Thanks a lot for this!

**Pull request merged**